exonum/src/api/node/public/explorer.rs

```diff
@@ -307,6 +306,7 @@ impl ExplorerApi {
     ) where
         Q: Fn(&HttpRequest) -> Result<SubscriptionType, ActixError> + Send + Sync + 'static,
     {
+        let max_message_len = calculate_message_len(&service_api_state);
```

Comment:
This is too strict; wrapping the transaction into `{ "type": "transaction", "payload": { "tx_body": tx } }` incurs additional overhead of >=45 bytes compared to the `calculated_message_len(..)`. We cannot specify an overhead with some leeway though (which I would otherwise recommend - a client implementation may insert some whitespace into JSON) because of the previously mentioned problem with WS processing; it swallows errors *and* does not check message size.